test/aggregators_tests.js

```diff
@@ -1,11 +1,9 @@
-/* global describe, it, before, beforeEach, after, afterEach */
```

Comment:
In the ESLint configuration, I set all the test files as using Mocha, so ESLint already knows these globals are present in test files.